src/main/java/org/ovirt/engine/extension/aaa/misc/mapping/MappingExtension.java

```diff
@@ -29,6 +29,28 @@
 import org.ovirt.engine.extension.aaa.misc.*;
```

Comment:
Please don't use wildcard import, better is to declare each required import separately

Reply:
The wildcard are not coming from my patch, they were already used.